aio/content/examples/testing/src/app/banner/banner-external.component.spec.ts

```diff
@@ -10,6 +10,21 @@ describe('BannerComponent (external files)', () => {
   let fixture: ComponentFixture<BannerComponent>;
   let h1: HTMLElement;
 
+  describe('setup that may fail', () => {
+    // #docregion setup-may-fail
+    beforeEach(async () => {
+      await TestBed.configureTestingModule({
+        declarations: [BannerComponent],
```

Comment:
It looks like the formatting in these files leave a space in front and right after a symbol, so I believe this should look like:
```suggestion
        declarations: [ BannerComponent ],
```

Reply:
@AndrewKushnir thanks for taking a look. Within `banner-external.component.spec.ts` there are a total of three `declarations: [BannerComponent]` statements (only one of which I introduced in this PR). Would you like me to change all three to `declarations: [ BannerComponent ]`?

Reply:
> Within banner-external.component.spec.ts there are a total of three declarations: [BannerComponent] statements (only one of which I introduced in this PR). Would you like me to change all three to declarations: [ BannerComponent ]?

Thanks for the comment. Let's keep it as is for now and see what @petebacondarwin suggests.

Reply:
The formatting of the examples is a bit ad hoc since we have not, so far, chosen or enforced a specific formatter tool.
Until we do that, we should try to keep the formatting consistent within each file. The other arrays in this file already contain spaces around the element, therefore I think that @AndrewKushnir is correct here.

Reply:
@petebacondarwin I made the suggested changes. thanks!